pkgs/shells/mksh/default.nix

```diff
@@ -1,26 +1,29 @@
 { stdenv, fetchurl }:
 
+with stdenv.lib;
```

Comment:
Not that it makes a difference, but why did you move this up here instead of keeping it down below, with what it's used for? It doesn't look like anything else needs `stdenv.lib` available...

Reply:
I prefer that way. So many times I needed all `stdenv.lib` functions readily available in other packages, then I started to declare it before everything.

Reply:
IMHO, that's not a valid reason to move it. No reason to give it a larger scope than necessary.

If this package ever needs `lib` functions readily available, you can add `lib` to the arguments list and use `lib.optional`, for example, instead of `stdenv.lib.optional`.